packages/contracts/hardhat.config.ts

```diff
@@ -65,8 +70,11 @@ const config: HardhatUserConfig = {
       throwOnTransactionFailures: true,
       throwOnCallFailures: true,
       blockGasLimit: 3000000000, // really high to test some things that are only possible with a higher block gas limit
-      gasPrice: 8000000000,
-      deploy: ['./deploy/new', './deploy/verification'],
+      gasPrice: 80000000000,
+      deploy:
+        process.env.TEST_UPDATE_DEPLOY_SCRIPT !== undefined
```

Comment:
The current example `.env` file will not work as expected with this flag

Reply:
if TEST_UPDATE_DEPLOY_SCRIPT is set to any value the feature will be activated